### ISSUE TITLE
ssh-bf are not detected when AllowUsers are defined

### ISSUE THREAD

**Issue by pfostenberg** [crowdsecurity/hub#1018]
**Describe the bug**
when in /etc/ssh/sshd_config a entry is made AllowUsers xyz the the bf filter will not detect bf attacks.

**To Reproduce**
Add a AllowUsers in the sshd_config try to login.
the you see a log like.
sshd[*]: User root from 1.2.3.4 not allowed because not listed in AllowUsers
but it will not be blocked.

**Expected behavior**
Also add this the to blocking

**Screenshots**
non

**Additional context**
improfe /etc/crowdsec/hub/parsers/s01-parse/crowdsecurity/sshd-logs.yaml will solve the problem
add pattern

```
# in case there are blocked by /etc/ssh/sshd_config AllowUsers xx yy
  SSHD_NOT_ALLOWED_USER: 'Failed password for invalid user\s*%{USERNAME:sshd_invalid_user}? from %{IP_WORKAROUND:sshd_client_ip}( port \d+)?'
```

and grok

```
- grok:
      name: "SSHD_NOT_ALLOWED_USER"
      apply_on: message
      statics:
        - meta: log_type
          value: ssh_failed-auth
        - meta: target_user
          expression: "evt.Parsed.sshd_invalid_user"
```

**Comment by LaurenceJJones** [comment]
Duplicate of #874 closing this issue and referencing it
